l10n_ar_account_check/models/account_check.py

```diff
@@ -490,6 +494,17 @@ def to_wallet(self):
             check.write(vals)
         return True
 
+    @api.multi
+    def check_to_wallet(self):
+        self.ensure_one()
+        if self.move_id:
+            self.move_id.button_cancel()
+            self.move_id.unlink()
+            self.move_id = False
+            self.state = 'wallet'
```

Comment:
@corsojulian7 Esto conviene hacerlo con un write.
```suggestion
            self.write({'state': 'wallet'})
```